### PR TITLE
feat: using custom lodash build

### PR DESCRIPTION
**Pull request by guypod**
- [X] Ready to merge
- [ ] Code review by @grnd 
## What does this PR do?

`lodash` is the biggest dependency of `snyk-resolve-deps`, though it only uses a couple of methods. 
This PR switches to using a custom build of `lodash`, using `lodash-cli`, which only extracts the `flatten` and `extend` functions, dramatically reducing its size.
## Related issue

https://snyksec.atlassian.net/browse/SC-1921

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,5 +49,14 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/Snyk/resolve-deps.git"
+  },
+  "nyc": {
+    "exclude": [
+      "dist",
```
Comment (guypod) [inline package.json:55]: the new dist folder was added, as it should be ignored from coverage.
All the rest are the default values, which now need to be stated.

File: package.json [hunk 2]
```diff
@@ -20,6 +21,8 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "jscs": "^2.8.0",
+    "lodash-cli": "^4.15.0",
```
Comment (guypod) [inline package.json:24]: Adds the `lodash` command used in the `build` script above

**Comment by cdaringe** [comment]
why not just import `lodash.methodname`, given that they are all individually published to npm?

**Comment by darscan** [comment]
Closing in favour of https://github.com/snyk/resolve-deps/pull/30 which allows de-duping in the CLI (but not yet, because the CLI currently uses the custom build approach)

**Pull request closed without merging**